css/style.css

```diff
@@ -1039,22 +1041,1372 @@ li { list-style-type: none; }
+    font-weight: 600;
+    font-size: 50px;
+    line-height: 61px;
+  }
+  
+  .time-speed__text {
+    display: flex;
+    align-items: flex-start;
+    font-size: 20px;
+    line-height: 24px;
+    font-style: normal;
+    font-weight: 600;
+    text-align: center;
+    width: 53.03px;
+    height: 35.83px;
+    margin: 22px 0 0 5px;
+    padding: 0;
+  }
+  
+  .subtext {
+    font-size: 12px;
+    line-height: 15px;
+    font-style: normal;
+    font-weight: normal;
+  }
+  
+  .time-speed__subtext {
+    display: flex;
+    justify-content: flex-start;
+    width: 110.35px;
+    height: 21.5px;
+    margin: -10px 0 0 86px;
+  }
+
+  .distance {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    align-content: flex-start;
+    width: 205px;
+    height: 83.12px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .distance__line {
+    display: flex;
+    width: 48px;
+    height: 80px;
+    margin: 0 13px 0 0;
+    background: url(https://i.ibb.co/ZWxz2h2/Road-icon-ipad.png) no-repeat;
+    background-size: 100% 90%;
+    background-position: right bottom;
+  }
+  
+  .distance__numeral {
+    display: flex;
+    width: 90.29px;
+    height: 61.62px;
+    margin: 0 12px 0 0;
+    padding: 0;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 50px;
+    line-height: 61px;
+  }
+  
+  .distance__text {
+    display: flex;
+    align-content: flex-start;
+    width: 41.56px;
+    height: 35.83px;
+    margin: 21.5px 0 0;
+    padding: 0;
+  }
+  
+  .distance__subtext {
+    display: flex;
+    width: 96.02px;
+    height: 21.5px;
+    font-style: normal;
+    font-weight: normal;
+    text-align: left;
+    margin: -22px 0 0 64px;
+    padding: 0;
+  }
+
+  .button {
+    display: flex;
+    width: 220.84px;
+    height: 55.21px;
+    font-family: "Montserrat", sans-serif;
+    text-align: center;
+    font-weight: bold;
+    text-transform: uppercase;
+  }
+  
+  .test-drive {
+    display: flex;
+    flex-flow: row wrap;
+    width: 418.21px;
+    height: 55.21px;
+    justify-content: flex-start;
+    align-items: center;
+    margin: 0 34.53px 6px 34.53px;
+  }
+
+  .test-drive__rudder {
+    display: flex;
+    width: 30.37px;
+    height: 30.37px;
+    margin: 0 9px 0 0;
+    padding: 0;
+    background: url(https://i.ibb.co/Stm3197/2-7-red-wheel.png) no-repeat;
+    background-size: 100%;
+    background-position: center;
+  }
+
+  .test-drive__link {
+    display: flex;
+    justify-content: center;
+    width: 122.84px;
+    height: 24.84px;
+    font-size: 16px;
+    line-height: 20px;
+    font-style: normal;
+    font-weight: bold;
+    text-decoration: none;
+    color: #e61234;
+    margin: 0 34.51px 0 0;
+    text-transform: uppercase;
+  }
+
+  .test-drive__link:hover {
+    display: flex;
+    font-size: 16px;
+    line-height: 20px;
+    font-weight: bold;
+    text-align: center;
+    text-decoration: none;
+    color: #000;
+    cursor: pointer;
+    padding: 0;
+    border: 0;
+  }  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5851407)